t/lib/bash/test_header

```diff
@@ -80,11 +80,40 @@
 #         a non-zero value.
 #     port_is_busy $PORT
 #         Return 0 if $PORT is busy or 1 if $PORT is not busy.
+#     rose_ws_init $NS $UTIL
+#         Start a Rose web service server. Test server OK. Write to these shell
+#         variables on success:
+#         * TEST_ROSE_WS_PID - PID of the service server
+#         * TEST_ROSE_WS_PORT - Port of the service server on localhost.
+#         * TEST_ROSE_WS_URL - URL of service server.
+#         E.g. rose_ws_init rose bush
+#     rose_ws_kill
+#         Kill a web service server started by "rose_ws_init" and remove
+#         generated log and status files.
+#     rose_ws_json_greps TEST_KEY JSON-FILE EXPECTED [EXPECTED ...]
+#         Load JSON content in argument 1
+#         Each remaining argument is an expected content in the JSON data in
+#         the form: [keys, value] where "keys" is a list containing keys or
+#         indexes to an expected item in the data structure, and the value is
+#         an expected value. A key in keys can be a simple dict key or an
+#         array index. It can also be a dict {attr_key: attr_value, ...}. In
+#         which case, the expected data item is under a list of dicts, where a
+#         unique dict in the list contains all elements attr_key: attr_value.
+#
+# VARIABLES
+#     TEST_DIR
+#         Tests will run in this temporary directory.
+#     TEST_KEY_BASE
+#         The base name without the "*.t" extension of the test file.
+#     TEST_NUMBER
+#         Number of tests already run.
+#     TEST_SOURCE_DIR
+#         The directory containing of the current test file.
 #-------------------------------------------------------------------------------
 set -eu
 
 SIGNALS="EXIT INT"
-function FINALLY() {
+FINALLY() {
```

Comment:
Yes. There are subtle differences between the 2 syntaxes, but it is not not necessary for any of our usages.